rocky/katalogus/templates/partials/plugin_tile.html

```diff
@@ -2,12 +2,19 @@
 {% load static %}
 
 <div role="group" id="plugin_{{ plugin.id|slugify }}">
+    {% if form_id %}
+        <input form="{{ form_id }}"
+               type="checkbox"
+               name="plugin"
+               value="{{ plugin.id }}"
+               {% if plugin.id in request.GET.urlencode %}checked{% endif %} />
```

Comment:
Just checking if the plugin.id exists in the complete query string seems wrong to me.

It looks like this is completely untested as we don't have any report with optional plugins at the moment. If I add an optional plugin  I get a checkbox seemingly random on the cat picture, but I have no idea what the purpose is as there is already an enable/disable button?

Reply:
Ahhh good one, I will change this.

Reply:
The optional plugins gets a checkbox to let the user decides whether they want to include this plugin to do scans that will result in the report.